resources/game_config.json

```diff
@@ -249,6 +249,10 @@
 	},
 
 
+	"lock_season": false, 
+	"_": "Forces the season to be locked at the clan's starting_season",
```

Comment:
can you change this to say comment instead of _ to be in line with how the comments on the other config options are?